pages/dkp/konvoy/1.7/prerequisites/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+navigationTitle: Prerequisites
+title: Prerequisites
```

Comment:
confusing as to why we have this now?

Reply:
This was merged a few months ago https://github.com/mesosphere/konvoy/pull/2951

Reply:
ok thanks for the context.